src/Resources/assets/styles/_tailwind.scss

```diff
@@ -36,13 +36,27 @@
         @apply text-base font-semibold text-neutral-500;
     }
 
+    .whatwedo-utility-content {
+        @extend .whatwedo-utility-paragraph;
+
+        a {
+            @apply text-base font-normal text-neutral-800;
```

Comment:
Assuming the link in general should have the same styling as the surrounding text, why not inherit all the text styles instead of setting them again? Or do we explicitly want this to not follow the style of the surrounding text at all?

Reply:
Good point. Absolutely this is duplicated. Removed the full line, can inherit indeed.